cishouseholds/pipeline/mapping.py

```diff
@@ -1434,6 +1434,158 @@
+        "PHM-FUP-SAM-023": "blood_return_date",
+        "PHM-FUP-SAM-024": "blood_return_future_date",
+        "PHM-FUP-SAM-025": "questionnaire_started_no_incentive",
+        "PHM-FUP-SAM-026": "test_kit_info_end_of_survey",
+        "PHM-FUP-WE-001": "aged_16_or_over",
+        "PHM-FUP-WE-002": "work_status_CRIS",
+        "PHM-FUP-WE-003": "work_status_employment",
+        "PHM-FUP-WE-004": "work_status_unemployment",
+        "PHM-FUP-WE-005": "work_status_education",
+        "PHM-FUP-WE-006": "work_in_additional_paid_employment",
+        "PHM-FUP-WE-007": "work_main_job_changed",
+        "PHM-FUP-WE-008": "work_main_job_title",
+        "PHM-FUP-WE-009": "work_main_job_role",
+        "PHM-FUP-WE-010": "work_sector",
+        "PHM-FUP-WE-011": "work_sector_other",
+        "PHM-FUP-WE-012": "work_health_care_area",
+        "PHM-FUP-WE-013": "work_nursing_or_residential_care_home",
+        "PHM-FUP-WE-014": "work_direct_contact_patients_or_clients",
+        "PHM-FUP-WE-015": "work_location",
+        "PHM-FUP-WE-016": "work_not_from_home_days_per_week",
+        "PHM-FUP-WE-017": "education_in_person_days_per_week",
+        "PHM-FUP-WE-018": "transport_to_work_or_education",
+        "PHM-FUP-WE-019": "ability_to_socially_distance_at_work_or_education",
+        "PHM-FUP-HST-001": "think_have_covid_any_symptom_list_1",
+        "PHM-FUP-HST-002": "think_have_covid_no_symptoms_list_",
+        # "PHM-FUP-HST-013": "confirm_symptoms_list_1",
+        "PHM-FUP-HST-003": "think_have_covid_any_symptom_list_2",
+        "PHM-FUP-HST-004": "think_have_covid_no_symptoms_list_2",
+        # "PHM-FUP-HST-023": "confirm_symptoms_list_2",
+        "PHM-FUP-HST-005": "think_have_symptoms_new_or_worse_list1",
+        "PHM-FUP-HST-006": "think_have_symptoms_new_or_worse_none_list1",
+        "PHM-FUP-HST-007": "think_have_symptoms_new_or_worse_list2",
+        "PHM-FUP-HST-008": "think_have_symptoms_new_or_worse_none_list2",
+        "PHM-FUP-HST-009": "think_have_covid_symptom_onset_date",
+        "PHM-FUP-HST-010": "illness_lasting_over_12_months",
+        "PHM-FUP-HST-011": "illness_reduces_activity_or_ability",
+        "PHM-FUP-HST-012": "ever_smoked_regularly",
+        "PHM-FUP-HST-013": "currently_smokes_or_vapes",
+        "PHM-FUP-HST-014": "currently_smokes_or_vapes_type",
+        "PHM-FUP-HST-015": "cis_covid_vaccine_received",
+        "PHM-FUP-HST-016": "cis_covid_vacine_type",
+        "PHM-FUP-HST-017": "cis_covid_vaccine_type_other",
+        "PHM-FUP-HST-018": "cis_covid_vaccine_number_of_doses",
+        "PHM-FUP-HST-019": "cis_covid_vaccine_date",
+        "PHM-FUP-HST-020": "flu_vaccination_received",
+        "PHM-FUP-HST-021": "flu_vaccine_date",
+        "PHM-FUP-HST-022": "been_outside_uk",
+        "PHM-FUP-HST-023": "been_outside_uk_last_country",
+        "PHM-FUP-HST-024": "been_outside_uk_last_return_date",
+        "PHM-FUP-C19-001": "PHM_think_had_respiratory_infection",
```

Comment:
Remove PHM_ from the following variable names: 
"PHM_think_had_respiratory_infection"
"PHM_think_had_respiratory_infection_type"
"PHM_think_had_flu_onset_date"
"PHM_think_had_other_infection_onset_date"